components/external_server/src/model/issue_tracker.py

```diff
@@ -183,6 +198,19 @@ def __get_field_options(self, issue_types: list[Option]) -> list[Option]:  # pra
             logging.warning("Getting issue tracker field options at %s failed: %s", api_url, reason)
         return [Option(str(field["fieldId"]), str(field["name"])) for field in fields]
 
+    def __get_epic_links(self, fields: list[Option]) -> list[Option]:  # pragma: no feature-test-cover
+        """Return the possible epic links for the current project."""
+        field_names = [field.name.lower() for field in fields]
+        if "epic link" not in field_names:
+            return []
+        api_url = self.epics_api % (self.url, self.issue_parameters.project_key)
+        epics = []
+        try:
+            epics = self.__get_json(api_url)["issues"]
+        except Exception as reason:  # pylint: disable=broad-except
+            logging.warning("Getting epics at %s failed: %s", api_url, reason)
+        return [Option(str(epic["key"]), str(f'{epic["fields"]["summary"]} ({epic["key"]})')) for epic in epics]
```

Comment:
Is there always a field summary? Is this the epic title/header?

Reply:
Yes, because we asked for the summary field via the `epics_api`.

Reply:
And it is the long version of the epic title. There is also a short version that Jira uses when it displays the epic as label, but it seemed more informative to use the long version in the epic link dropdown menu. The dropdown menu also shows the epic key by the way. Also, the epic summary is a mandatory field in Jira: when you create a new epic you have to enter both a short title and a summary.